### DISCUSSION TITLE
Development roadmap and schedule

### DISCUSSION THREAD

**Discussion by MRSSSMR** [KevinBytesTheDust/CVAmp#185] [Q&A]
Hello jlplenio!
Is there a new update schedule
and........... May I know what updates are waiting for?

I haven't heard from you for about a month, so I'm very curious haha

And I don't know if it works from the 0.7.0 version or the previous version
What's clear is that the 0.7.0 version has become a raid feature

**Comment by jlplenio** [comment]
Hey @MRSSSMR. I am at capacity currently with my day job. But what's on top of my list is: 
- Add a new dedicated tab to the GUI for chatting and automatic chatting settings, like timers.
- Chat messages loaded from file and posted automatically via time interval.
- Persistent pairs of auth_tokens and proxies.
- Verification, per instance, that the chat functionality is established and working.
- Option to choose between spawning of a full or light functionality instance (to possibly reduce workload).

I updated the todo list here https://github.com/jlplenio/crude-twitch-viewer-bot/discussions/177

Daily, supporters on ko-fi are reporting valuable information, suggestions, and issues. That is really nice to see.
